open_event/api/sessions.py

```diff
@@ -0,0 +1,72 @@
+from flask.ext.restplus import Resource, Namespace, fields
+
+from open_event.models.session import Session as SessionModel
+from open_event.models.event import Event as EventModel
+from .helpers import get_object_list, get_object_or_404, get_object_in_event
+
+api = Namespace('sessions', description='Sessions', path='/')
+
+_track = api.model('Track', {
+    'id': fields.Integer(required=True),
+    'name': fields.String,
+})
+
+_speaker = api.model('Speaker', {
+    'id': fields.Integer(required=True),
+    'name': fields.String,
+})
+
+_level = api.model('Level', {
+    'id': fields.Integer(required=True),
+    'label_en': fields.String,
+})
+
+_language = api.model('Language', {
+    'id': fields.Integer(required=True),
+    'label_en': fields.String,
+    'label_de': fields.String,
+})
+
+_microlocation = api.model('Microlocation', {
+    'id': fields.Integer(required=True),
+    'name': fields.String,
+})
+
+_session = api.model('Session', {
+    'id': fields.Integer(required=True),
+    'title': fields.String,
+    'subtitle': fields.String,
+    'abstract': fields.String,
+    'description': fields.String,
+    'start_time': fields.DateTime,
+    'end_time': fields.DateTime,
+    'track': fields.Nested(_track),
+    'speakers': fields.List(fields.Nested(_speaker)),
+    'level': fields.Nested(_level),
+    'language': fields.Nested(_language),
+    'microlocation': fields.Nested(_microlocation),
+})
+
+
+@api.route('/events/<int:event_id>/sessions/<int:session_id>')
+@api.response(404, 'Session not found')
+@api.response(400, 'Object does not belong to event')
+class Session(Resource):
+    @api.doc('get_session')
+    @api.marshal_with(_session)
+    def get(self, event_id, session_id):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2620995762/issues/source?bid=3233864&fileBranchId=3314204#l57)